bin/run-tests.ps1

```diff
@@ -1,5 +1,5 @@
 $tmpRoot = [System.IO.Path]::GetTempPath()
-$exercises = Get-ChildItem -Path exercises/practice -Directory
+$exercises = Get-ChildItem -Path ../exercises/practice -Directory
```

Comment:
I think the idea is that one runs `pwsh ./bin/run-tests.ps1`, in which case this path is incorrect. Maybe we should make this work in both cases and do:

```suggestion
$exercises = Get-ChildItem -Path $PSScriptRoot/../exercises/practice -Directory
```

Reply:
That is a much better solution. I will update it.